### PR TITLE
change some styles

### PR DESCRIPTION
**Pull request by Diane27**
here are the changes made
- Change green to dvc green
- Change secondary color to match the darker green
- Make questions on faqs orange
- Add logo in place of DVC HACKS text
- Fix typo on sponsor caption
- Remove about section
- Added link to registration typeform

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -217,27 +204,27 @@ <h1>Venue</h1>
       <h1>FAQ</h1>
       <div class="grid-x grid-margin-x">
           <div class="cell medium-6 large-4">
-            <h3>What is a hackathon?</h3>
+            <h3 class="question">What is a hackathon?</h3>
             <p>A hackathon is best described as an “invention marathon”. Anyone who has an interest in technology attends a hackathon to learn, build &amp; share their creations over the course of a weekend in a relaxed and welcoming atmosphere. You don’t have to be a programmer and you certainly don’t have to be majoring in Computer Science.</p>
           </div>
           <div class="cell medium-6 large-4">
-            <h3>Who can attend?</h3>
+            <h3 class="question">Who can attend?</h3>
```
Comment (tejashah88) [inline index.html:211]: You could mention that it's open to high school and college students

File: index.html [hunk 2]
```diff
@@ -217,27 +204,27 @@ <h1>Venue</h1>
...
           </div>
           <div class="cell medium-6 large-4">
-            <h3>What if I don’t have a team or idea?</h3>
+            <h3 class="question">What if I don’t have a team or idea?</h3>
             <p>Don&#x27;t worry! We’ll have team-forming activity to help you find teammates. Some of the best hacks usually come from teams that met and formed an idea on the spot!</p>
           </div>
           <div class="cell medium-6 large-4">
-            <h3>What if I don’t know how to code?</h3>
+            <h3 class="question">What if I don’t know how to code?</h3>
             <p>Not a problem! The entire organizing team knew very little when they each attended their very first hackathon. It’s entirely irrelevant what your experience is going into a hackathon, it’s more about your interest in technology. DVCHacks is made to be very welcoming and beginner-friendly. We will provide workshops and mentors to help you build something you will be proud of</p>
```
Comment (tejashah88) [inline index.html:220]: Error 404: Ending period not found

File: src/partials/landing.html [hunk 3]
```diff
@@ -1,17 +1,14 @@
...
+    <img src="img/dvchacks-logo-white.png" class="logo">
+
+    <p class="caption">
       Come hack with us and compete for prizes!
     </p>
     <div class="divider"></div>
     <div class="event-info">
       <p>April 13-14</p>
-      <p>12:00AM - 12:00AM</p>
+      <p>9:00AM - 8:00PM & 9:00AM - 2:00PM </p>
```
Comment (tejashah88) [inline src/partials/landing.html]: Might be a good idea to indicate which day is for which time, like "Saturday on 9 AM - 8 PM" and "Sunday on 9 AM - 2 PM"

**Review by tejashah88 — COMMENTED** [review]
Just a few content change suggestions but looks good overall.

File: src/partials/landing.html (hunk 3 above)
Comment (Diane27) [reply src/partials/landing.html]: I added it at first but it looked a bit too long. I'll put it back and we'll see how it looks.

**Pull request merged**